cmd/daemon/flux/exporter.go

```diff
@@ -28,18 +22,19 @@ type Exporter interface {
 }
 type ReleaseManagerExporter struct {
 	Log         *log.Logger
-	Url         string
-	AuthToken   string
 	Environment string
+	Client      httpinternal.Client
 }
 
 func (f *ReleaseManagerExporter) Send(_ context.Context, event event.Event) error {
 	f.Log.With("event", event).Infof("flux event logged")
-	client := &http.Client{
-		Timeout: 20 * time.Second,
+	var resp httpinternal.FluxNotifyResponse
+	url, err := f.Client.URL("webhook/daemon/flux")
+	if err != nil {
+		return err
 	}
-	b := &bytes.Buffer{}
-	err := json.NewEncoder(b).Encode(httpinternal.FluxNotifyRequest{
+	f.Log.Infof("URL: %s", url)
```

Comment:
Is this required to get logged? To me it looks a bit debuggy?

Reply:
It was a debug log line. It's removed 👍